### PR TITLE
Update git slides

### PR DESCRIPTION
**Pull request by danielinteractive**

### CODE REVIEW

File: slides/05_collaboration.qmd [hunk 1]
```diff
@@ -69,8 +69,8 @@ Photo from <a href="https://unsplash.com/@timmossholder?utm_source=unsplash&utm_
 flowchart LR
   A[<font size=4> Kevin v1] --> B[<font size=4> Kevin v2]
   B --> C[<font size=4> Kevin v3]
```
Comment (danielinteractive) [inline slides/05_collaboration.qmd]: @shuangli22 probably makes more sense to replace "Kevin" by "Shuang" and then replace "Daniel" by your partner for the demonstration, e.g. Joe or Liming

**Review by shajoezhu — APPROVED** [review]
lgtm! Thanks @danielinteractive

**Comment by danielinteractive** [comment]
thanks Joe, I think we still need some updates on the names, see above comment

File: slides/05_collaboration.qmd [hunk 2]
```diff
@@ -319,23 +319,23 @@ gitGraph
 sequenceDiagram
```
Comment (clarkliming) [inline slides/05_collaboration.qmd:319]: in the presentation html the text could not be read clearly

Comment (clarkliming) [inline slides/05_collaboration.qmd:319]: in the presentation html the text could not be read clearly
image

Comment (danielinteractive) [reply slides/05_collaboration.qmd:319]: is there box size option? maybe we can also make a first pass now and see how it renders on the github pages site, locally can look different

File: slides/05_collaboration.qmd [hunk 3]
```diff
@@ -69,8 +69,8 @@ Photo from <a href="https://unsplash.com/@timmossholder?utm_source=unsplash&utm_
 flowchart LR
   A[<font size=4> Kevin v1] --> B[<font size=4> Kevin v2]
   B --> C[<font size=4> Kevin v3]
-  B --> D[<font size=4> Daniel v1]
-  D --> E[<font size=4> Kevin + Daniel v4]
+  B --> D[<font size=4> Shuang v1]
+  D --> E[<font size=4> Kevin + Shuang v4]
   C --> E
```
Comment (clarkliming) [inline slides/05_collaboration.qmd:74]: in line 95, suggest to use `SVN` to replace `SCM` (svn is version control tool, similar to git but scm is much bigger)

File: slides/05_collaboration.qmd (hunk 2 above)
Comment (shuangli22) [reply slides/05_collaboration.qmd:319]: It looks okay in my end
image

Comment (danielinteractive) [reply slides/05_collaboration.qmd:319]: then no worries

Comment (shuangli22) [reply slides/05_collaboration.qmd:319]: @clarkliming How you generated that? It looks good in my output
image

File: slides/05_collaboration.qmd (hunk 3 above)
Comment (danielinteractive) [reply slides/05_collaboration.qmd:74]: +1

File: slides/05_collaboration.qmd (hunk 2 above)
Comment (clarkliming) [reply slides/05_collaboration.qmd:319]: probably this is a duplicated message as I said the same here https://github.com/openpharma/workshop-r-swe/pull/10#discussion_r1144185343

File: slides/05_collaboration.qmd (hunk 3 above)
Comment (shuangli22) [reply slides/05_collaboration.qmd:74]: Thank you, updated

File: slides/05_collaboration.qmd (hunk 1 above)
Comment (shuangli22) [reply slides/05_collaboration.qmd]: @danielinteractive Thank you, updated.
Hi @shajoezhu, I updated the partner name to you in the git workflow diagram. Maybe later in the demo, I will request the merge request review from you, is that okay?

Comment (shajoezhu) [reply slides/05_collaboration.qmd]: sure. no problem! we can practice tomorrow!

**Pull request merged**